homeassistant/components/fitbit/sensor.py

```diff
@@ -472,6 +472,11 @@ def device_state_attributes(self):
     def update(self):
         """Get the latest data from the Fitbit API and update the states."""
         if self.resource_type == "devices/battery" and self.extra:
+            new_dev_status = self.client.get_devices()
```

Comment:
I would change the variable name to stay consistent with line 281.
```suggestion
            registered_devs = self.client.get_devices()
```